config/authorities/linked_data/oclc_fast.json

```diff
@@ -1,15 +1,13 @@
 {
   "QA_CONFIG_VERSION": "2.1",
   "prefixes": {
-    "dcterms": "http://purl.org/dc/terms/",
-    "skos": "http://www.w3.org/2004/02/skos/core#",
-    "schema": "http://schema.org/"
```

Comment:
`skos` and `schema` prefixes are defined in the `ldpath` gem.  So they are not required here.

NOTE: In `ldpath` gem, `schema.org` prefix is `rdfs`